test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -270,9 +270,44 @@ Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)"
         $result | should be ([char]0x1b)
     }
 
-	It "Test that escaping any character with no special meaning just returns that char. (line 602)" {
-        $result = ExecuteCommand '"fo`obar"'
-		$result | should be "foobar"

```

Comment:
Good catch! Thanks.  Was debugging utf8 encoding problems with the file and removed this one by accident.